readthedocs.yml

```
@@ -0,0 +1,18 @@
+
+version: 2
+
+sphinx:
+  fail_on_warning: false
+
+formats:
+  - pdf
+  - epub
+
+python:
+  version: 3.8
+  system_packages: false
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

```suggestion
        - docs

```

Please add an empty line at the end of each file... it will make future diff easier to read.

Thanks!